src/containers/App.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import BookList from './BookList';
+import BooksForm from '../components/BooksForm';
+import { createBook, deleteBook } from '../actions';
+
+const App = ({ books }) => (
```

Comment:
Our App component must be in the src/components directory! Let us fix it :smile: